.rubocop_todo.yml

```diff
@@ -248,13 +257,40 @@ Naming/VariableName:
   Exclude:
     - 'app/models/description.rb'
 
+# Offense count: 1
+# Cop supports --auto-correct.
+Performance/RegexpMatch:
+  Exclude:
+    - 'app/classes/robots.rb'
+
+# Offense count: 18
+# Cop supports --auto-correct.
+Performance/Size:
+  Exclude:
+    - 'app/views/api2/_collection_number.xml.builder'
+    - 'app/views/api2/_herbarium_record.xml.builder'
+    - 'app/views/api2/_image.xml.builder'
+    - 'app/views/api2/_location.xml.builder'
+    - 'app/views/api2/_name.xml.builder'
+    - 'app/views/api2/_observation.xml.builder'
+    - 'app/views/api2/_project.xml.builder'
+    - 'app/views/api2/_species_list.xml.builder'
+    - 'app/views/api2/_user.xml.builder'
+
+# Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: AutoCorrect.
+Performance/StartWith:
+  Exclude:
+    - 'app/classes/robots.rb'
+
```

Comment:
@pellaea Not an issue for this PR.
But if you want to make these files marginally more performant, have a look at these cops. (Or if you like, I'll do it. That could be a better use of your time.)

Reply:
I've been meaning to fix this.  I thought rubocop preferred count to size.  Apparently, I just got the two names mixed up.  I used to think they were all synonyms.  I'll take care of it tonight.